bin/replace

```diff
@@ -9,4 +9,4 @@ shift
 replace_with="$1"
 shift
 
-ag -l --nocolor "$find_this" $* | xargs sed -i '' "s/$find_this/$replace_with/g"
+ag -l --nocolor "$find_this" "$@" | xargs sed -i '' "s/$find_this/$replace_with/g"
```

Comment:
@gabebw This and the corresponding quoting in `g()` were the two changes I was most concerned about. I was unable to break them in my personal testing, but wasn't sure if there's nuance I'm missing.